ProcessMaker/Http/Controllers/Api/InboxRulesController.php

```diff
@@ -23,8 +23,9 @@ public function index(Request $request)
         $order_direction = $request->input('order_direction', 'desc');
         $per_page = $request->input('per_page', 10);
         $filter = $request->input('filter', '');
+        $user_id = $request->input('user_id', 'userId');
 
-        $query = InboxRule::query();
+        $query = InboxRule::query()->where('user_id', $user_id);
```

Comment:
@CarliPinell can we use the current logged in user? `$request->user()->id`?

Reply:
You're right Nolan, I'll change that code.